README.md

```diff
@@ -1,6 +1,6 @@
 # Abtion Guidelines
 
-Please, go to the specific folder for more info.
+Please, go to the specific folders for more info.
```

Comment:
Why `folders`? It is a specific folder the one you'll go to.

Reply:
Because it is at the root of the project, where there are the javascript, rails and wordpress folders.
As in there are several to choose from.
Basing it purely on gut feeling though :X

Reply:
It's true there are several `folders` but you go to one `folder`. You can't go to `folders` :)

Reply:
yes you can! ... just gotta be... quick... :(